andela/blackninjas-frontend
# #169817846 Implement chat functionality

#### What does this PR do?
It implements a chat app
#### Description of Task to be completed?
- Send public message
#### How should this be manually tested?
- Log into barefoot nomad and click on the message icon you should see a chat component where you can write and send a message
#### Any background context you want to provide?
- this feature is still in development
#### What are the relevant pivotal tracker stories?
#### Screenshots (if appropriate)
#### Questions:
